src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -509,7 +509,7 @@ private static ImmutableArray<CompletionItemWithHighlight> GetHighlightedList(Li
             ImmutableArray<CompletionFilterWithState> filters)
         {
             // See which filters might be enabled based on the typed code
-            var _ = PooledHashSet<CompletionFilter>.GetInstance(out var textFilteredFilters);
+            using var _ = PooledHashSet<CompletionFilter>.GetInstance(out var textFilteredFilters);
```

Comment:
Shouldn't CA2000 catch such cases? Is it disabled for the repo?

Reply:
I never trusted CA2000 because it can't figure ownership transport correctly.

Reply:
Yes, CA2000 is disabled for the repository.